ads/yieldbot.js

```diff
@@ -32,32 +33,39 @@ export function yieldbot(global, data) {
 
   loadScript(global, 'https://cdn.yldbt.com/js/yieldbot.intent.js', () => {
     global.ybotq.push(() => {
-      const dimensions = [[
-        parseInt(data.overrideWidth || data.width, 10),
-        parseInt(data.overrideHeight || data.height, 10),
-      ]];
+      try {
+        const dimensions = [[
+          parseInt(data.overrideWidth || data.width, 10),
+          parseInt(data.overrideHeight || data.height, 10),
+        ]];
 
-      global.yieldbot.psn(data.psn);
-      global.yieldbot.enableAsync();
-      if (window.context.isMaster) {
-        global.yieldbot.defineSlot(data.ybSlot, {sizes: dimensions});
-        global.yieldbot.go();
-      } else {
-        let slots = {};
-        slots[data.ybSlot] = dimensions;
-        global.yieldbot.nextPageview(slots);
+        global.yieldbot.psn(data.psn);
+        global.yieldbot.enableAsync();
+        if (window.context.isMaster) {
+          global.yieldbot.defineSlot(data.ybSlot, {sizes: dimensions});
+          global.yieldbot.go();
+        } else {
+          const slots = {};
+          slots[data.ybSlot] = dimensions;
+          global.yieldbot.nextPageview(slots);
+        }
+      } catch (e) {
+        rethrowAsync(e);
       }
     });
 
     global.ybotq.push(() => {
-      let targeting = global.yieldbot.getSlotCriteria(data['ybSlot']);
-      data['targeting'] = data['targeting'] || {};
-      for (var key in targeting) {
-        data.targeting[key] = targeting[key];
+      try {
+        const targeting = global.yieldbot.getSlotCriteria(data['ybSlot']);
+        data['targeting'] = data['targeting'] || {};
+        for (const key in targeting) {
+          data.targeting[key] = targeting[key];
+        }
+      } catch (e) {
+        rethrowAsync(e);
```

Comment:
I don't really care. While we do have our own error reporting in the ad frame, we only log syncronously during initial load. Otherwise it is too noisy for us.